payments/tests/conftest.py

```diff
@@ -1,17 +1,22 @@
 import pytest
+from factory.random import randgen
 
 from berth_reservations.tests.conftest import *  # noqa
 from payments.tests.factories import (
     AdditionalProductFactory,
-    BerthPriceGroupFactory,
     BerthProductFactory,
     WinterStorageProductFactory,
 )
+from resources.tests.conftest import *  # noqa
+from resources.tests.factories import BerthTypeFactory
 
 
 @pytest.fixture
 def berth_price_group():
-    berth_price_group = BerthPriceGroupFactory()
+    width = randgen.uniform(1, 999)
```

Comment:
some comment why we are doing it this way would be welcome here for future generations 🙂